lib/fizzy/parser.hpp

```diff
@@ -17,6 +17,11 @@ constexpr bytes_view wasm_prefix{wasm_prefix_data, sizeof(wasm_prefix_data)};
 template <typename T>
 using parser_result = std::pair<T, const uint8_t*>;
 
+/// Parses `input` into a Module.
+///
+/// @param input The WebAssembly binary. No need to persist by the caller, since all relevant parts
```

Comment:
Did you want 4 spaces between argument name and comment?

Reply:
I'll copy your style from #664.